### PR TITLE
Deposits update

### PR DESCRIPTION
**Pull request by Gadjung**
#185 - make deposits up-to date with other minerals

will fix also Novice Walls not being displayed.

everything that has 0 as max HP will show Indestructible instead of 0/0

Deposits:
image
image
image
image

NoviceWall:
image

### CODE REVIEW

**Comment by thmsndk** [comment]
@Gadjung Is there a screenshot of novice walls? I kinda wanted to do them "connected" so it was an area, and not individual boxes, so i'm kinda curious how it looks :)

**Comment by Gadjung** [comment]
it's same model as constructed walls, just with `fixed` height and Indestructible in hits panel

**Comment by Gadjung** [comment]
for the connected part it's the same issue as with other 'nice to be connected things' like ramparts or ordinary constructed walls (though less issue due to uniform height

File: Assets/Scripts/Screeps3D/Tools/Selection/Subpanels/HitpointsPanel.cs [hunk 1]
```diff
@@ -39,7 +39,11 @@ public override bool IsPanelAvailabelForObject(RoomObject roomObject)
         private void UpdateLabel()
         {
             _meter.SetVisibility(_hitsObject.Hits / _hitsObject.HitsMax);
-            _label.text = string.Format("{0:n0} / {1:n0}", _hitsObject.Hits, (long) _hitsObject.HitsMax);
+            if((long) _hitsObject.HitsMax > 0) {
+                _label.text = string.Format("{0:n0} / {1:n0}", _hitsObject.Hits, (long) _hitsObject.HitsMax);
+            } else {
+                _label.text = string.Format("Indestructible");
```
Comment (thmsndk) [inline Assets/Scripts/Screeps3D/Tools/Selection/Subpanels/HitpointsPanel.cs:45]: We could color the text if we wanted to by writing "<color=red>indistructibe</color>" at least I think that is how it is. i've done it different places. what do you think?

**Comment by thmsndk** [comment]
Oh it's just the walls? I was referring to safenode it is called zone border cause I intended to show novice and respawn zones that way ☺

**Comment by Gadjung** [comment]
this is just the novice zone walls that were not displayed due to scale being set to 0

**Comment by thmsndk** [comment]
What happens to the terrain when the deposit decays / disappears?

**Comment by Gadjung** [comment]
nothing if room is viewed and deposit disappears, on revisiting / reloading room @Robalian script should make walls default according to script (not lowered to 0.3f)

**Review by thmsndk — APPROVED** [review]

**Pull request merged**